### DISCUSSION TITLE
Where can i text the github support ?

### DISCUSSION THREAD

**Discussion by saifeddineelhanoune** [community/community#114337] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

even i have answered more than two discussion on github community i'm not able to earn the galaxy brain achievement

**Comment by tantd2203** [comment]
))) why do you have to achieve the galaxy I think you should support the community.

**Comment by tantd2203** [comment]
u can flow https://www.youtube.com/watch?v=oJWFfp3sGYA <3

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @0x013FA11 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. 

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

**Reply by saifeddineelhanoune** [reply]
i'm thrilled to have received a response from a GitHub staff member regarding my inquiry about the Galaxy Brain achievement
My intention was always to engage genuinely with the community sharing my knowledge and learning from others I'm disappointed to hear about the changes as I was looking forward to earning the Galaxy Brain achievement through real interactions and contributions However I understand the reasons behind this decision and am committed to continuing my journey on GitHub by contributing meaningfully to discussions and projects
Thank you for your assistance and for helping me understand the new guidelines

**Comment by Tgentil** [comment]
To earn the Galaxy Brain achievement on GitHub, it's not just about responding questions; your answers must be marked as the solution to the question in discussions ( two times) . A quick look at your GitHub activity suggests that only one of your responses has been marked as the answer to a question. Keep contributing meaningfully to the discussions, and as more of your answers are recognized as solutions, you should be able to earn the achievement.

Your response marked as a Answer:

![Your response marked as a Answer](https://github.com/community/community/assets/88681055/ae4c1c83-24eb-45fd-bba0-21f3ac5ba581)

For a brief guide on how to earn the Galaxy Brain achievement, you can read this: [How do I get the Galaxy Brain badge?](https://github.com/orgs/community/discussions/18293) for more insights on effective participation.

**Reply by saifeddineelhanoune** [reply]
yes the maintainers of the discussions i answered have marked my answers as the solution of their context
i always gives answers to the community and rarely when someone mark my answers but with time and with more asks and answers i got marked but i don't how have just seen one response that have been marked

**Reply by Tgentil** [reply]
@0x013FA11, it sounds like you're actively contributing valuable answers in discussions. I only checked the responses marked as solutions in the GitHub Community discussions. It's possible you've provided solutions in other organizations or repositories that I didn't review. To check this, you can view your activity graph on your GitHub profile, filter by 'GitHub Community', and select the relevant year to see all your contributions. Here's  [ayour activity graph](https://github.com/0x013FA11). Keep contributing, and soon you may see more of your answers being recognized as solutions, leading you to the Galaxy Brain achievement.

![your contributions](https://github.com/community/community/assets/88681055/56bf1fbd-b920-4739-ac0e-59b8ffd3cd15)

from the image, you can see that you initiated two discussions in the GitHub Community and one of your responses was marked as an answer. However, please note that GitHub has made changes to the achievements system, especially regarding the Community discussions. For more details on these changes and how they might affect the achievement process, you can check the discussion on [GitHub Community achievements update](https://github.com/orgs/community/discussions/106536). Keep up your contributions in various repositories, as your answers in different contexts might also contribute to achieving the Galaxy Brain status.

**Reply by saifeddineelhanoune** [reply]
thank you for sharing all this informations with me brother i appreciate this and this will make me more interactive in the community and could help others
